### PR TITLE
Router and basic structure

### PR DESCRIPTION
**Pull request by ark2002**
- Added Router and links for routing
- Added Placeholders for Screens
- Did the Barrel exports as suggested in the previous PR

### CODE REVIEW

File: src/frontend/components/Banners/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export { QouteBanner } from "./QouteBanner/QouteBanner"
```
Comment (MarkVed17) [inline src/frontend/components/Banners/index.js:1]: check for spelling err

Comment (MarkVed17) [reply src/frontend/components/Banners/index.js:1]: Quote*

**Review by MarkVed17 — APPROVED** [review]
The structure looks proper Aryak.
Just check for spelling errors & code formatting.

File: src/frontend/components/Banners/index.js (hunk 1 above)
Comment (ark2002) [reply src/frontend/components/Banners/index.js:1]: yup ill look for it

File: src/frontend/components/Banners/TitleBanner/TitleBanner.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import "./TitleBanner.css";
+import { Link } from "react-router-dom"
```
Comment (akash-1318) [inline src/frontend/components/Banners/TitleBanner/TitleBanner.js:3]: You can also import Link in same way as you are importing Router from frontend file.

**Review by akash-1318 — COMMENTED** [review]
Just added one suggestion look into that else everything looks good from my end.

**Pull request merged**